### PR TITLE
a few fixes for insights deploys

### PR DESCRIPTION
**Pull request by melvinsoft**
The PR contains some minor fixes for Insights:
- Change mysql for mysql_init on `eucalyptus_insights_single.yml` playbook.
- Add MySQL option for Pipeline Input and Output databases.
- Add Tag to Create OAuth keys tasks so we can avoid to run the entire edxapp playbook.
- Minor fixes for our `setup_pipeline` playbook.

### CODE REVIEW

**Comment by melvinsoft** [comment]
[(6) HIOA Insights](https://trello.com/c/GhaqzWBF/1038-6-hioa-insights)

File: playbooks/roles/setup_pipeline/tasks/main.yml [hunk 1]
```diff
@@ -131,3 +131,8 @@
   become_user: "{{ pipeline_user }}"
   tags:
     - install
+
+- name: Adding hadoop user to pipeline group
```
Comment (mrgnr) [inline playbooks/roles/setup_pipeline/tasks/main.yml]: @melvinsoft Why is this needed? For mysql TLS certs?

Comment (melvinsoft) [inline playbooks/roles/setup_pipeline/tasks/main.yml]: @mrgnr No, we need this, because hadoop needs to create some dirs on this folder while the pipeline is running.
